api/core/settings.py

```diff
@@ -104,8 +105,12 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'api.management.middlewares.set_secure_headers',
+    'corsheaders.middleware.CorsMiddleware',
 ]
 
+ALLOWED_HOSTS=['*']
```

Comment:
This is very dangerous. Using `'*'` as allowed hosts is only OK for Local development, I can fix this in this branch if that is ok for you @blacky-yg ?